src/app/lib/analyticsUtils/index.js

```diff
@@ -10,24 +10,36 @@ import {
   XTOR_CAMPAIGN_IDENTIFIER,
   SUPPORTED_MEDIUM_CAMPAIGN_TYPES,
 } from './analytics.const';
-
-export const getAmpDestination = (psDestination, gnlDestination) => {
-  /*
-   ** AMP Variable substitution is used here to construct a string that
-   ** will get the correct destination according to the client's geolocation,
-   ** found using <amp-geo>.
-   ** https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md
-   ** https://amp.dev/documentation/components/amp-geo/
-   */
-
-  // eslint-disable-next-line no-template-curly-in-string
-  const ampGeo = '${ampGeo}'; // String representing the list of matched groups (comma delimited) e.g. eea,gbOrUnknown
-  const withGbOrUnknownMatched = `$MATCH(${ampGeo}, gbOrUnknown, 0)`; // Checks for presence of 'gbOrUnknown' and returns 'gbOrUnknown' if found
-  const equalsgbOrUnknown = `$EQUALS(${withGbOrUnknownMatched}, gbOrUnknown)`; // Returns 'true' if the result of the $MATCH was 'gbOrUnknown'
-  return `$IF(${equalsgbOrUnknown}, ${psDestination}, ${gnlDestination})`; // If 'true', use PS destination, otherwise use GNL
-};
+import getAmpDestination from './getAmpDestination';
 
 export const getDestination = (platform, statsDestination) => {
+  const geoVariants = {
+    NEWS_PS: {
+      PS: 598285,
+      GNL: 598287,
+    },
+    NEWS_PS_TEST: {
+      PS: 598286,
+      GNL: 598288,
+    },
+    SPORT_PS: {
+      PS: 598310,
+      GNL: 598308,
+    },
+    SPORT_PS_TEST: {
+      PS: 598311,
+      GNL: 598309,
+    },
+    NEWS_LANGUAGES_PS: {
+      PS: 598291,
+      GNL: 598289,
+    },
+    NEWS_LANGUAGES_PS_TEST: {
+      PS: 598292,
+      GNL: 598290,
```

Comment:
to make this a little clearer, could you move `destinationIDs` before this list an reference the mapped detinations in under the `PS` and `GNL` keys e.g.
```
    NEWS_LANGUAGES_PS: {
      PS: destinationIds.NEWS_LANGUAGES_PS,
      GNL: destinationIds.NEWS_LANGUAGES_GNL,
    },
```
Just makes it less likely there have been no copy/paste mistakes too